### ISSUE TITLE
Visual bug when you are orbiting a sun

### ISSUE THREAD

**Issue by moeck2** [Eggs-D-Studios/wos-issues#126]
### Guidelines

- [X] I checked for duplicate bug reports
- [X] I tried to find a way to reproduce the bug

### Version

Main (Stable)

### What happened? What did you expect to happen?

Whenever im orbiting a sun (coordinates X, X, 0, 0) a visual bug keeps appearing whenever i look at the right angle (probably being the sun). Not sure what really causes this (but someone did point out that a dev could've messed with the spawn point or made the stars too big so that we're inside them, so maybe check that.) and it happens to most stars (I've went to about 5 stars and 4 had the bug), and there doesnt seem to be a criteria for when it happens either (or at least from what ive seen)

https://github.com/user-attachments/assets/ef290b3f-2191-4a90-a234-7e490b77b36c

### How can we reproduce this bug?

Just enter a sun's orbit
Or
1. Find a solar system
2. Go to said solar system
3. Orbit the sun (make sure the last two numbers in the coordinates are 0, 0)

### Frequency

Most of the time

**Comment by hail12pink** [comment]
yeah this is a thing. fixed (along with star models themselves being fixed) in an outdated pr that i need to update when i can.
